apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -493,7 +493,9 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
 
                 const protocolName = mapProtocolTypeToName(protocol.type);
 
-                protocolNames.push(protocolName);
+                if(!protocolNames.includes(protocolName)){
+                    protocolNames.push(protocolName);
+                }
 
                 inboundProtocolRequests.push(getInboundProtocolConfig(appId, protocolName));
```

Comment:
Don't we need to skip this `inboundProtocolRequests.push(getInboundProtocolConfig(appId, protocolName));` as well if we got duplicate protocols?

Reply:
Yes, had uncommited changes, sorry for the inconvenience, resolving this comment now.